cypress.json

```diff
@@ -1,4 +1,4 @@
 {
   "video": false,
-  "baseUrl": "http://127.0.0.1"
+  "baseUrl": "http://127.0.0.1:3000"
```

Comment:
I guess there wont be any problem to merge this. If so we do not need to change it locally.